bin/bootstrax

```diff
@@ -455,24 +460,24 @@ def now(plus=0):
     return datetime.now(pytz.utc) + timedelta(seconds=plus)
 
 
-def kill_process(pid, wait_time=None):
-    """Kill process pid, or raise RuntimeError if we cannot
-    :param wait_time: time to wait before escalating signal strength
-    """
-    if wait_time is None:
-        wait_time = timeouts['signal_escalate']
+def kill_process(pid):
+    """Kill process pid"""
+    log.warning(f'Kill PID:{pid}')
     if not pid_exists(pid):
+        log.warning(f'No PID:{pid}')
         return
 
-    for sig in [signal.SIGTERM, signal.SIGKILL, 'die']:
-        time.sleep(wait_time)
-        if not pid_exists(pid):
-            return
-        if signal == 'die':
-            message = f"Could not kill process {pid}"
-            log_warning(message, priority='fatal')
-            raise RuntimeError(message)
-        os.kill(pid, sig)
+    parent = Process(pid)
+    for child in parent.children(recursive=True):
+        child.kill()
+    parent.kill()
+
+    # Just make it extra dead
+    os.kill(pid, signal.SIGKILL)
+
+    if pid_exists(pid):
+        message = f"Could not kill process {pid}?!"
+        log_warning(message, priority='fatal')        
```

Comment:
Old version only killed the parent - resulting in many child processes waiting to die